docs/data/joy/components/menu/MenuIconSideNavExample.js

```diff
@@ -98,15 +97,21 @@ function MenuButton({ children, menu, open, onOpen, onLeaveMenu, label, ...props
         placement: 'right-start',
         sx: {
           width: 288,
-          [`& .${menuClasses.listbox}`]: {
```

Comment:
Shouldn't be deleted. You can add these to the matching TS demo and run `yarn docs:typescript:formatted`.

Reply:
problem is menu classes does not have the listbox property

`const menuClasses: MenuClasses = generateUtilityClasses('MuiMenu', [
  'root',
  'expanded',
  'colorPrimary',
  'colorNeutral',
  'colorDanger',
  'colorInfo',
  'colorSuccess',
  'colorWarning',
  'colorContext',
  'variantPlain',
  'variantOutlined',
  'variantSoft',
  'variantSolid',
  'sizeSm',
  'sizeMd',
  'sizeLg',
]);
`

Reply:
You are right. I opened a new PR to address this issue. After I get a review from another team member on the PR, I will approve this one and merge it. Thanks.

Reply:
This issue is now fixed in #36520